usaspending_api/broker/management/commands/load_fsrs.py

```diff
@@ -499,7 +499,7 @@ def cleanup_broken_links(self, db_cursor):
 
     @transaction.atomic
     def handle(self, *args, **options):
-        logger.info('Starting FSRS data load...')
```

Comment:
You seen very opinionated on the "correct" log messages

Reply:
It's less strong opinion and more low cost. These scripts are pretty much doing the same thing, so I feel they should look the same wherever it's cheap to do so.